VHostScan/lib/helpers/color_helper.py

```diff
@@ -0,0 +1,104 @@
+from termcolor import colored
+
+__COLOR_ENABLED = True
+
+text_settings = {
+    'process':      ('cyan', []),
+    'result_head':  ('magenta', []),
+    'result':       ('white', []),
+    'key':          ('yellow', []),
+    'link':         ('green', ['underline']),
+    'error':        ('red', ['bold']),
+    'warning':      ('red', ['bold', 'dark']),
+}
```

Comment:
Since we are already keeping a dictionary of types to colors/attributes why don't we get rid of the termcolor dependency? That will be one less thing that users will have to install.

What you can do is go ahead and make a mapping of the colors that you want to use to their term codes and store that in all_colors by making it a dictionary. 
ie `all_colors = {'blue':'\x1b[34'}` 
Then in your __convert_to_color function, print the mapped color + value + white. that's essentially all that termcolor is doing.

Thoughts @codingo ?

Reply:
I would like to do so but I believe that termcolor has its mechanism dealing platform incapabilities as well as further problems beyond my current knowledge, and termcolor is not an opensource too so it's not very convenient to learn about how they did it.

I feel that doing so is reinventing a large and complex wheel, but as I said I only 'feel' so.
If you know that there is no such terrible things, please let me know, I will give it a try ^^

Reply:
I just found sourcecode of termcolor. Will look through it roughly first finding some clues to answer what's in my mind.

Reply:
I ran through it already thinking this is interesting but it will require sometime lol

Reply:
I will let you know when I made an update on this